tests/_config/kuttl-test.yaml

```diff
@@ -4,7 +4,5 @@ startKIND: false
 kindConfig: "./kind-config.yaml"
 parallel: 1
 timeout: 300
-testDirs:
-- tests/e2e/v1beta2/setup
```

Comment:
No need to remove this. 
The e2e test case would override this `testDirs` since we have passed that through CLI.